__tests__/utils/CairoTypes/uint256.test.ts

```diff
@@ -167,3 +175,334 @@ describe('CairoUint256 class test', () => {
     ]);
   });
 });
+
+describe('Utility Functions', () => {
+  describe('isBigInt', () => {
+    test('should return true for big integers', () => {
+      expect(isBigInt(BigInt(10))).toBe(true);
+      expect(isBigInt(BigInt('9007199254740991'))).toBe(true);
+    });
+
+    test('should return false for non-big integers', () => {
+      expect(isBigInt(10)).toBe(false);
+      expect(isBigInt('10')).toBe(false);
+      expect(isBigInt(undefined)).toBe(false);
+      expect(isBigInt(null)).toBe(false);
+      expect(isBigInt({})).toBe(false);
+      expect(isBigInt([])).toBe(false);
+      expect(isBigInt(true)).toBe(false);
+    });
+  });
+
+  describe('isString', () => {
```

Comment:
maybe we should move these tests to other test files, like `utils.test.ts` or `shortString`, whatever makes more sense!